packages/background/src/chains/handler.ts

```diff
@@ -30,6 +31,8 @@ export const getHandler: (service: ChainsService) => Handler = (service) => {
           env,
           msg as RemoveSuggestedChainInfoMsg
         );
+      case ChangeChainMsg:
```

Comment:
Not sure about this one from looking at `ChainInfo` implementations/usages but do we want to be consistent and use the same `ChainInfo` terminology here? The handled message has a `chainInfo` property, thus why I'm wondering it might make more sense